src/services/registrationData.ts

```diff
@@ -74,3 +74,81 @@ export async function overwriteRegistrationData({
   // return new registration data
   return newRegistrationData;
 }
+
+export async function updateQuestionOptions(
+  // inserts new question options or updates existing ones
```

Comment:
make this comment the tsdoc of this function

Reply:
updated function ts-doc